### PR TITLE
Tablefilters

### PR DESCRIPTION
**Pull request by gcnabeel**
This branch includes table filters and onboarding application fixes

### CODE REVIEW

File: Datahub.Core/Components/FormTemplate.razor [hunk 1]
```diff
@@ -1,7 +1,11 @@
 ﻿@inject IStringLocalizer DataLocalizer
+@using Microsoft.EntityFrameworkCore
 @using Datahub.Core.Data
 @typeparam T
+@typeparam U where U : DbContext
+@inject UIControlsService UI
 @inherits HtmlElement
+@inject IDatahubAuditingService AuditingService
```
Comment (gcerik) [inline Datahub.Core/Components/FormTemplate.razor:8]: is this used yet?

File: WebPortal/Pages/Forms/Finance/Budget.razor [hunk 2]
```diff
@@ -13,7 +13,7 @@
 <div>
     <AeFlex Vertical id="indexFlex">
         <AePaginatedTable Accessors="accessorFunctions"
-                 T="global::Datahub.Portal.Data.Finance.Budget"
+                 T="Datahub.Portal.Data.Finance.Budget"
```
Comment (gcerik) [inline WebPortal/Pages/Forms/Finance/Budget.razor:16]: the finance module should move to its own project like the language one

**Review by gcerik — APPROVED** [review]
we need these change but I added few suggestions

File: Datahub.Core/Components/FormTemplate.razor (hunk 1 above)
Comment (gcnabeel) [reply Datahub.Core/Components/FormTemplate.razor:8]: Yes

**Pull request merged**